assets/app/scripts/services/data.js

```diff
@@ -630,6 +630,15 @@ angular.module('openshiftConsole')
   };
 
   DataService.prototype._listOpComplete = function(type, context, data) {
+    // Here we normalize all items to have a kind property.
+    // One of the warts in the kubernetes REST API is that items retrieved
+    // via GET on a list resource won't have a kind property set.
```

Comment:
Can you add the link to the k8s issue open to address that:
https://github.com/kubernetes/kubernetes/issues/3030